google-cloud-bigquery/lib/google/cloud/bigquery/data.rb

```diff
@@ -292,7 +292,7 @@ def dml?
         # @return [String, nil] The DDL operation performed.
         #
         def ddl_operation_performed
-          return nil unless job_gapi && job_gapi.statistics.query
+          return nil unless job_gapi&.statistics&.query
           job_gapi.statistics.query.ddl_operation_performed
```

Comment:
Both lines can be replaced with `job_gapi&.statistics&.query&.ddl_operation_performed`

Similar for the next two modifications.